src/algorithms/expected_consensus.md

```diff
@@ -292,10 +339,16 @@ w[r+1] = w[r] + floor(1000 * (wForkFactor(wPowerFactor[r+1] + wBlocksFactor[r+1]
 
```

Comment:
Read other comments first

---

Given everything else I would define `w[r+1]` as:
`w[r+1] = w[r] + (wForkFactor_num * (wPowerFactor[r+1] + wBlocksFactor[r+1) * 2^16) / wForkFactor_den`

`2^16` can be replaced by `2^8` depending on precision required.